addons/mod_tool/dock.tscn

```diff
@@ -503,14 +505,69 @@ script = ExtResource( 2 )
 [node name="ValidationDelay" type="Timer" parent="TabContainer/Default Configuration/VBox/ConfigEditor"]
 one_shot = true
 
+[node name="Popup" type="WindowDialog" parent="."]
+unique_name_in_owner = true
+visible = true
+anchor_left = 0.5
+anchor_top = 0.5
+anchor_right = 0.5
+anchor_bottom = 0.5
+margin_left = -200.0
+margin_top = -75.0
+margin_right = 200.0
+margin_bottom = 75.0
+window_title = "Create new mod"
```

Comment:
needs some min_size, otherwise the window becomes tiny and unreadable for me

Reply:
Made it resizable and gave it a min size (350, 150) 👍